MendelaA/HW6/2.py

```diff
@@ -0,0 +1,14 @@
+# Task2. Write a script that checks the login that the user enters.
+# If the login is "First", then greet the users. If the login is
+# different, send an error message.
+# (need to use loop while)
+
```

Comment:
good job
look at this code:

while True:
    login = input('Enter your login: ')
    if login == 'First':
        print('Greetings!')
        break
    else:
        print('Error message!\nTry again.')